src/logic/publish.utils.ts

```diff
@@ -46,19 +47,17 @@ export function prePublishCheck(
   try {
     parseProviderJson(providerJson);
   } catch (error) {
-    //TODO: better way of formating?
-    if ('issues' in error) {
-      for (const issue of (error as { issues: [] }).issues) {
-        if ('path' in issue && 'message' in issue) {
-          result.push({
-            kind: 'error',
-            message: `Provider check error: ${
-              (issue as { message: string }).message
-            } on path ${(issue as { path: string }).path}`,
-          });
-        }
+    if (error instanceof ZodError)
```

Comment:
i meant just some simple typeguard, don't add the zod dependency which is going to be removed from sdk soon anyway